src/ai/backend/manager/dto/request.py

```diff
@@ -22,3 +24,41 @@ class SearchArtifactsReq(BaseRequestModel):
     # TODO: Support this. (we need to make strawberry independent types)
     # ordering: Optional[ArtifactOrderingOptions] = None
     # filters: Optional[ArtifactFilterOptions] = None
+
+
+class ScanArtifactModelsReq(BaseRequestModel):
+    models: list[ModelTarget] = Field(description="List of models to scan from the registry.")
+    registry_id: Optional[uuid.UUID] = Field(
+        default=None, description="The unique identifier of the artifact registry to scan."
+    )
+
+
+class CleanupArtifactsReq(BaseRequestModel):
+    artifact_revision_ids: list[uuid.UUID] = Field(
+        description="List of artifact revision IDs to cleanup."
+    )
+
+
+class CancelImportArtifactReq(BaseRequestModel):
+    artifact_revision_id: uuid.UUID = Field(
+        description="The artifact revision ID to cancel import."
+    )
+
+
+class ApproveArtifactRevisionReq(BaseRequestModel):
+    artifact_revision_id: uuid.UUID = Field(description="The artifact revision ID to approve.")
+
+
+class RejectArtifactRevisionReq(BaseRequestModel):
+    artifact_revision_id: uuid.UUID = Field(description="The artifact revision ID to reject.")
+
+
+class ImportArtifactsReq(BaseRequestModel):
+    artifact_revision_ids: list[uuid.UUID] = Field(
+        description="List of artifact revision IDs to import."
+    )
+
+
+class UpdateArtifactReq(BaseRequestModel):
+    artifact_id: uuid.UUID = Field(description="The artifact ID to update.")
+    modifier: ArtifactModifier = Field(description="Fields to modify in the artifact.")
```

Comment:
It seems good to create a separate field because the modifier is intended for internal processing objects. Please make it have fields similar to gql.